test-data/poly_template/poly_eval.py.j2

```diff
@@ -1,14 +1,14 @@
 #!/usr/bin/env python
 
+import datetime
 import json
+import os
 
-import datetime
+import ecl_data_io as eclio
+import numpy
+from numpy import array
```

Comment:
Use `numpy.array`?

Reply:
Hmm, this file I didn't touch, but probably `isort` touched it :)  Fixed!